doc/cassandra_compatibility.md

```diff
@@ -117,17 +127,18 @@ The following fields are optional in Cassandra 4.0 and may be missing in some re
-| host          | COORDINATOR | C* 4.0 will print \<IP\>:\<port\>, while ecAudit only print \<IP\>                     |
-| source        | CLIENT      | C* 4.0 will print \[\<hostname\>\]/\<IP\> (including the '/'), while ecAudit only print \<IP\> |
-| port          | -           | Not present in ecAudit, addressed in [#90](https://github.com/Ericsson/ecaudit/issues/90) |
-| batch         | BATCH_ID    | Correlation ID for entries in separated batches                                        |
-| timestamp     | TIMESTAMP   |                                                                                        |
-| type          | -           | Not present in ecAudit                                                                 |
-| category      | -           | Not present in ecAudit                                                                 |
-| ks            | -           | Not present in ecAudit                                                                 |
-| scope         | -           | Not present in ecAudit                                                                 |
-| operation     | OPERATION   |                                                                                        |
-| -             | STATUS      | Not present in Cassandra                                                               |
+| Cassandra 4.0 | ecAudit         | Comment |
+| ------------- | --------------- | -------------------------------------------------------------------------------------- |
+| user          | USER            |                                                                                        |
+| host          | COORDINATOR     | C* 4.0 will print \<IP\>:\<port\>, while ecAudit only print \<IP\>                     |
+| source        | CLIENT          | C* 4.0 will print \[\<hostname\>\]/\<IP\> (including the '/'), while ecAudit only print \<IP\> |
+| port          | -               | Not present in ecAudit, addressed in [#90](https://github.com/Ericsson/ecaudit/issues/90) |
+| batch         | BATCH_ID        | Correlation ID for entries in separated batches                                        |
+| timestamp     | TIMESTAMP       |                                                                                        |
+| type          | -               | Not present in ecAudit                                                                 |
+| category      | -               | Not present in ecAudit                                                                 |
+| ks            | -               | Not present in ecAudit                                                                 |
+| scope         | -               | Not present in ecAudit                                                                 |
+| operation     | OPERATION_NAKED | operation, without bound values appended to prepared statements                        |
```

Comment:
Added "OPERATION_NAKED" which corresponds to C4.0 "operation" - without prepared statement values...